registry/registry.go

```diff
@@ -36,8 +37,7 @@ type RegDebugInfo struct {
 // Service stores a Service constructor (mandatory) and an attribute converter.
 type Service struct {
 	RegDebugInfo
-	Constructor           CreateService
-	AttributeMapConverter config.AttributeMapConverter
```

Comment:
We register attribute map converters independently now, so this was just driveby cleanup.